src/test/java/com/fasterxml/jackson/core/tofix/LocationOfError1180Test.java

```diff
@@ -195,4 +199,18 @@ public String toString()
         public final int lineNr;
         public final int columnNr;
     }
+
+    public static class ShouldPredicate1180Test
+            implements ExpectedPassingTestCasePredicate
+    {
+        @Override
+        public boolean shouldPass(List<Object> arguments) {
```

Comment:
I am not so sure about this -- seems bit fragile and not very easy to understand. Is something like this expected for all "tofix" parameterized tests?

Reply:
Oh. Looks like this is just one case, maybe there is something special about that one.